website/templates/emails/no_login.html.mako

```diff
@@ -0,0 +1,30 @@
+## -*- coding: utf-8 -*-
+<%inherit file="notify_base.mako"/>
+<%def name="content()">
+    <div style="margin: 40px;">
+        <br>
+        Hello ${fullname},
+        <br>
```

Comment:
Please add an empty line after the greeting: 
![screen shot 2015-09-21 at 12 52 09 pm](https://cloud.githubusercontent.com/assets/6414394/9999411/4af067ca-6064-11e5-867e-07216fd87958.png)